src/components/Menus/MainMenu/useMainMenuContent.tsx

```diff
@@ -70,7 +69,7 @@ export const useMainMenuContent = () => {
 
   let mainMenu: any[] = [];
 
-  if (configTheme?.hasThemeModeSwitch) {
+  if (configTheme?.hasThemeModeSwitch && !disableThemeSwitch) {
```

Comment:
won't be enough as the dark theme does not exist so we have to force the light theme on this page